broker/src/main/java/io/camunda/zeebe/broker/system/partitions/impl/steps/LogStoragePartitionTransitionStep.java

```diff
@@ -140,12 +148,4 @@ public void appendEntry(
               lowestPosition, highestPosition));
     }
   }
-
-  public static final class LogStorageTermMissmatchException
-      extends RecoverablePartitionTransitionException {
-    private LogStorageTermMissmatchException(
-        final long expectedTerm, final long actualTerm, final int partitionId) {
-      super(String.format(WRONG_TERM_ERROR_MSG, expectedTerm, actualTerm, partitionId));
```

Comment:
:question: Do I have a wrong config? Seems I moved some methods in some classes

Reply:
Not sure, that sometimes happens to me too (although not in a long time). Maybe @korthout or @remcowesterhoud have some ideas?

Reply:
Or we haven't updated all classes yet? :man_shrugging: